docs/options.md

```diff
@@ -673,10 +673,13 @@ Platform-specific environment variables are also available:<br/>
 !!! note
     cibuildwheel always defines the environment variable `CIBUILDWHEEL=1`. This can be useful for [building wheels with optional extensions](faq.md#building-packages-with-optional-c-extensions).
 
+!!! note
+    To do its work, cibuildwheel internally sets the options `VIRTUALENV_PIP`, `DIST_EXTRA_CONFIG`, `SETUPTOOLS_EXT_SUFFIX`, `PIP_DISABLE_PIP_VERSION_CHECK`, `PIP_ROOT_USER_ACTION`. Your assignments to these options might be overridden.
```

Comment:
@joerick looks like `PIP_CONSTRAINT` is missing from this list.

Reply:
My intention was that it's not here because we don't override it, we extend it.

Reply:
Then, maybe it's worth noting that it ends up being not exactly what the end-user sets?